config-templates/azure/azuredisk-csi-driver/1.23.0/deployment.yaml

```diff
@@ -0,0 +1,667 @@
+---
+apiVersion: v1
+kind: List
+metadata:
+  name: azuredisk-csi-driver
+  namespace: kube-system
+  annotations:
+    version: 1.23.0
+    revision: 1
+items:
+  - apiVersion: v1
+    stringData:
+      cloud-config: |
+        {
+          "cloud":"AzurePublicCloud",
+          "tenantId": "{{{tenantId}}}",
+          "subscriptionId": "{{{subscriptionId}}}",
+          "aadClientId": "{{{aadClientId}}}",
+          "aadClientSecret": "{{{aadClientSecret}}}",
+          "resourceGroup": "{{{resourceGroup}}}",
+          "location": "{{{location}}}",
+          "vmType": "{{{vmType}}}",
+          "securityGroupName": "{{{securityGroupName}}}",
+          "vnetName": "{{{vnetName}}}"
+        }
+    kind: Secret
+    metadata:
+      name: azure-cloud-provider
+      namespace: kube-system
+    type: Opaque
+  - apiVersion: storage.k8s.io/v1
+    kind: CSIDriver
+    metadata:
+      name: disk.csi.azure.com
+      annotations:
```

Comment:
Following labels are not added in this version similar to previous verson.
https://github.com/IBM/ibm-satellite-storage/blob/master/config-templates/azure/azuredisk-csi-driver/1.18.0/deployment.yaml#L35-L37